subsys/bluetooth/mesh/light_ctrl_srv.c

```diff
@@ -376,8 +376,15 @@ static uint32_t centilux_get(struct bt_mesh_light_ctrl_srv *srv)
 
 static void transition_start(struct bt_mesh_light_ctrl_srv *srv,
 			     enum bt_mesh_light_ctrl_srv_state state,
-			     uint32_t fade_time)
+			     uint32_t fade_time,
+			     bool scene_recall)
 {
+	if (IS_ENABLED(CONFIG_BT_MESH_SCENE_SRV)) {
+		if (!scene_recall) {
+			bt_mesh_scene_invalidate(srv->model);
```

Comment:
Why not call this from `scene_recall`?

Reply:
Have you thought of the side effects this will create specially when device is already in current scene and nothing else is happening?

Reply:
The only place where this is called is from `scene_recall`, or am I missing something?

Reply:
Ah, I see, it is not `scene_recall`!